axum/src/extract/multipart.rs

```diff
@@ -142,6 +142,38 @@ impl<'a> Field<'a> {
     pub async fn text(self) -> Result<String, MultipartError> {
         self.inner.text().await.map_err(MultipartError::from_multer)
     }
+
+    /// Stream a chunk of the field data.
+    ///
+    /// When the field data has been exhausted, this will return [`None`].
+    ///
+    /// This does the exact same thing as the `Stream` impl and is for convenience
```

Comment:
I think this reads a bit nicer

```suggestion
    /// Note this does the same thing as `Field`'s [`Stream`] implementation.
```

Reply:
Agreed